src/helm/benchmark/tokenizer_config_registry.py

```diff
@@ -0,0 +1,57 @@
+import os
+from typing import Dict, Optional, List
+from dataclasses import dataclass
+
+import cattrs
+import yaml
+
+from helm.common.hierarchical_logger import hlog
+from helm.common.object_spec import ObjectSpec
+
+
+TOKENIEZR_CONFIGS_FILE = "tokenizer_configs.yaml"
+
+
+class TokenizerSpec(ObjectSpec):
+    pass
+
+
+@dataclass(frozen=True)
+class TokenizerConfig:
+    """Configuration for a tokenizer."""
+
+    name: str
+    """Name of the tokenizer."""
+
+    tokenizer_spec: TokenizerSpec
```

Comment:
This is `tokenizer_spec` instead of `client_spec` because I think that eventually Tokenizers and Clients should be separate classes...